### PR TITLE
Solved the UI issues in Navigation Setting / Server #15873

### LINKED ISSUE #15873: Unexpected UI in Navigation Setting / Server
### To Reproduce

1. Go to 'Navigation Panel'
2. Click on 'Setting (⚙️) icon '
3. Go to 'Server' tab

### Expected behavior

image

### Screenshots

image

### Additional context

Only in Master-branch

### PR DESCRIPTION
**Pull request by OfficialOzioma**
Signed Off By-Ozioma Dike <pascalozioma18@gmail.com>

### Description

Solved the UI issues in sittings/navigation panel>server server, Setting/Main Panel 

git
Fixes #15873

### CODE REVIEW

**Comment by OfficialOzioma** [comment]
@williamdes is my PR ready for Merge

File: exit [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  [31mremotes/origin/MAINT_4_8_0[m
+  [31mremotes/origin/QA_4_6[m
+  [31mremotes/origin/QA_4_7[m
+  [31mremotes/origin/QA_4_8[m
+  [31mremotes/origin/QA_4_9[m
+  [31mremotes/origin/QA_5_0[m
+  [31mremotes/origin/STABLE[m
+  [31mremotes/origin/gsoc-js-refactoring[m
+  [31mremotes/origin/gsoc-theme-generator[m
+  [31mremotes/origin/master[m
```
Comment (williamdes) [inline exit]: Please remove this file

File: exit [hunk 2]
```diff
@@ -0,0 +1,29 @@
+  dev[m
```
Comment (williamdes) [inline exit]: And this one

File: libraries/classes/Config/FormDisplay.php [hunk 3]
```diff
@@ -514,7 +514,7 @@ private function _displayFieldInput(
         );
     }
 
-    /**
+    /**
```
Comment (williamdes) [inline libraries/classes/Config/FormDisplay.php]: This change is strange

File: libraries/entry_points/preferences/forms.php [hunk 4]
```diff
@@ -19,7 +19,7 @@
 
 if (! defined('PHPMYADMIN')) {
     exit;
-}
+}
```
Comment (williamdes) [inline libraries/entry_points/preferences/forms.php]: Same here

File: templates/home/index.twig [hunk 5]
```diff
@@ -12,6 +12,7 @@
     <div class="row">
       <div class="col-7 col-xs-12">
         {% if has_server %}
+
```
Comment (williamdes) [inline templates/home/index.twig]: Same

**Review by williamdes — CHANGES_REQUESTED** [review]
Can you remove the unrelated changes?

**Comment by yashrajbothra** [comment]
IMO You have to undo all the commits and unstage those changes which had to be deleted.

and aslo I recommend you to use VS Code 👍

**Comment by williamdes** [comment]
@OfficialOzioma I think solution would be
```
git reset --hard origin/master
# do the change once more
# commit
git push --force
```

**Comment by OfficialOzioma** [comment]
> @OfficialOzioma I think solution would be
> 
> ```
> git reset --hard origin/master
> # do the change once more
> # commit
> git push --force
> ```

**Comment by OfficialOzioma** [comment]
@williamdes Thanks am working on it

**Pull request closed without merging**